

const vulnerabilityService = require('../Services/mysql/vulnerabilityService')


module.exports.getPoamsByVulnerabilityId = async function getPoamsByVulnerabilityId(req, res, next) {
    try {
        var poams = await vulnerabilityService.getPoamsByVulnerabilityId(req, res, next);
            res.status(200).json(poams);
    } catch (error) {
        if (error.status === 400) {
            res.status(400).json({ error: 'Validation Error', detail: error.errors });
        } else {
            res.status(500).json({ error: 'Internal Server Error', detail: error.message });
        }
    }
}

module.exports.getExistingVulnerabilityPoams = async function getExistingVulnerabilityPoams(req, res, next) {
    try {
        var existingPoams = await vulnerabilityService.getExistingVulnerabilityPoams(req, res, next);
            res.status(200).json(existingPoams);

    } catch (error) {
        res.status(500).json({ error: 'Internal Server Error', detail: error.message });
    }
}